src/codeu/chat/server/Server.java

```diff
@@ -39,8 +39,13 @@
 import codeu.chat.util.Timeline;
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.Connection;
+import codeu.chat.server.Model;
+import codeu.chat.server.Snapshotter;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
 
-public final class Server {
+import java.io.Serializable;
+public final class Server implements Serializable {
```

Comment:
The Model, and everything in it, needs to be Serializable, not the Server class.